www/source/javascripts/community.js

```diff
@@ -0,0 +1,25 @@
+$(function() {
+  if ($(".community-events")) {
+    var url = DEPOT_API + "/community-events";
+
+    $.ajax(url, {
+      success: function(data, status, xhr) {
+        if (typeof data === "object" && typeof data.length === "number") {
```

Comment:
You're right, it's probably always going to be an array, but on the off chance something weird ends up in there, I'd rather it fail gracefully.  Thanks for the tip about `Array.isArray`.  That's totally what I wanted.  I'll update it.